setup.py

```diff
@@ -53,15 +53,15 @@
         "Development Status :: 3 - Alpha",
         "Intended Audience :: Developers",
         "Intended Audience :: Science/Research",
-        "License :: OSI Approved :: MIT License",
+        "License :: OSI Approved :: Apache v2.0",
```

Comment:
This line should be `License :: OSI Approved :: Apache Software License`. Unfortunately, there is no v2 (and [these](https://github.com/pypa/pypi-legacy/issues/91) [issues](https://github.com/pypa/trove-classifiers/issues/17) note that this needs to be clarified). Small point, but want to make sure things are indexed correctly in `pypi`.